### PR TITLE
Adeed trigger for helm build

### PR DESCRIPTION
**Pull request by jaydee029**
For issue galasa-dev/projectmanagement#1935

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        name: trigger helm workflow
+        runs-on: ubuntu-latest
+        permissions: write-all
+
+        steps:
+        - name: Triggering workflow using Github API call
+          run: |
+           gh workflow run build-helm.yaml --repo https://github.com/${{env.NAMESPACE}}/automation
+          env:
+            GH_TOKEN: ${{ github.token }}
```
Comment (jadecarino) [inline .github/workflows/build.yaml:21]: Please change this to `${{ secrets.GALASA_TEAM_GITHUB_TOKEN }}` I believe this is needed for cross repo triggering, where `github.token` doesn't have the necessary permissions.

**Review by jadecarino — CHANGES_REQUESTED** [review]
Just one small change - other than that looks good

File: .github/workflows/build.yaml (hunk 1 above)
Comment (jaydee029) [reply .github/workflows/build.yaml]: done

Comment (jaydee029) [reply .github/workflows/build.yaml]: probably the reason this was failing.

**Review by jadecarino — APPROVED** [review]
Approved - I'll merge on your behalf when the Galasa 0.36.0 release is done

**Pull request merged**